### PR TITLE
Update requirements.md

### PR DESCRIPTION
**Pull request by scnbwy**
Logitech BRIO NOT Support firmware versions V240，only support V2.2.50，logitech support URL：https://support.logi.com/hc/en-us/articles/360024703494

### CODE REVIEW

**Comment by scnbwy** [comment]
show all download,Only V2.2.50 versions will be found
image

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/room-systems/requirements.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ These devices are available at [aka.ms/teamsdevices](https://aka.ms/teamsdevices
 |Microsoft Teams Rooms peripheral|Certified firmware version | Camera supports content camera use|
 |:--- |:--- | :--- |
 |[Crestron Huddly IQ](https://www.crestron.com/en-US/Products/Workspace-Solutions/Unified-Communications/Crestron-Flex-Accessories/CCS-CAM-USB-F-400)   | 1.02.09.33901  | &#x2714; |
-|[Logitech BRIO](https://www.logitech.com/en-us/product/brio)   |v240| &#x2714; |
+|[Logitech BRIO](https://www.logitech.com/en-us/product/brio)   |v240（Logitech NOT Support firmware versions V240，only support V2.2.50，logitech support URL：https://support.logi.com/hc/en-us/articles/360024703494）| &#x2714; |
```
Comment (JohanFreelancer9) [inline Teams/room-systems/requirements.md]:
```suggestion
|[Logitech Brio](https://www.logitech.com/en-us/product/brio)   |V2.2.50（Logitech does NOT Support firmware version V240）| &#x2714; |
```

Comment (scnbwy) [reply Teams/room-systems/requirements.md]: Thank you for edit

**Comment by scanum** [comment]
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by LanaChin** [comment]
@scnbwy, @JohanFreelancer9 - I updated the change to just reflect the version number in the table.  Thanks.

**Pull request merged**